smoke test fail case: [global aggr on changelog_kv stream with single shard, checkpoint]

<!-- You have to provide the following information whenever possible. -->

**Describe what's wrong**
https://github.com/timeplus-io/proton/actions/runs/8994490983/job/24708056140#step:7:23387
```
2024-05-08T01:16:11.0019141Z test_stream_smoke/test_query_smoke.py:68: AssertionError
2024-05-08T01:16:11.0019925Z ------------------------------ Captured log call -------------------------------
2024-05-08T01:16:11.0022407Z INFO     test_query_smoke:test_query_smoke.py:66 expected_results = [{'query_id': '15113', 'expected_results': [['[1, 2]'], ['[1, 3]']]}, {'query_id': '15113-1', 'expected_results': [['[1, 3, 4, 5]'], ['[4, 5]'], ['[5]']]}], statements_results_designed = [{'query_id': '15113', 'query_result': [['[1, 2]'], ['[1, 3]']]}, {'query_id': '15113-1', 'query_result': [['[1, 3]'], ['[1, 3, 4, 5]'], ['[4, 5]'], ['[5]']]}]
2024-05-08T01:16:11.0024994Z - generated html file: file:///actions-runner/_work/proton/proton/tests/stream/report_nativelog_2024-05-08 00:59:08.049916.html -
2024-05-08T01:16:11.0026101Z =========================== short test summary info ============================
2024-05-08T01:16:11.0027441Z FAILED test_stream_smoke/test_query_smoke.py::test_run[changelog_stream14-113-global aggr on changelog_kv stream with single shard, checkpoint]
2024-05-08T01:16:11.0028673Z ================== 1 failed, 839 passed in 1021.11s (0:17:01) ==================
2024-05-08T01:16:11.0029359Z setting = nativelog, setting_test_duration = 1021
```
**How to reproduce**

is this flaky test?

--- a/tests/stream/test_stream_smoke/0013_changelog_stream14.yaml
+++ b/tests/stream/test_stream_smoke/0013_changelog_stream14.yaml
@@ -255,6 +255,7 @@ tests:
       - changelog_kv
       - single shard
       - checkpoint
+      - bug
     name: "global aggr on changelog_kv stream with single shard, checkpoint"
     description: global aggr on changelog_kv stream with single shard, checkpoint
     steps:

